src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,115 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double SIZE_MULTIPLICATION = 1.5;
+    private int arrayLength = DEFAULT_CAPACITY;
+    private T[] array = (T[]) new Object[DEFAULT_CAPACITY];
```

Comment:
let's initialise this field in the constructor

Reply:
done